generators/app/templates/org.xtext.example.mydsl.web/build.gradle

```diff
@@ -1,6 +1,14 @@
 plugins {
```

Comment:
There IS a dep to ace , there should be a dependency to cm or non at all ( see dependencies section could not a a comment there from phone )